<launch>

<arg name="cam1_name" default="cam1"/>
<arg name="cam1_dev" value="/dev/video2"/>

<group ns="aruco1">
  <node name="$(arg cam1_name)" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="$(arg cam1_dev)"/>
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="pixel_format" value="mjpeg" />
    <param name="color_format" value="yuv422p" />
    <param name="framerate" value="30" />
    <param name="camera_frame_id" value="$(arg cam1_name)" />
    <param name="io_method" value="mmap"/>
    <param name="camera_info_url" value="package://sisyph_slam/config/hp_720p_30fps.yaml"/>
  </node>
</group>

</launch>
